onto_spread_ed/release/GithubPublishReleaseStep.py

```
@@ -17,10 +17,12 @@ def run(self) -> bool:
                              config.main_branch)
         self._raise_if_canceled()
 
-        files = [f.target.file for f in self._release_script.files.values()]
+        artifacts = self.artifacts()
+
+        files = [a.target_path for a in artifacts]
         self._total_items = len(files)
         for index, file in enumerate(files):
```

Ruff

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)